win32/tests/sysinfo_win32_ut/sysinfo_win32_ut.c

```diff
@@ -46,7 +46,9 @@ TEST_SUITE_INITIALIZE(suite_init)
     ASSERT_ARE_EQUAL(int, 0, umock_c_init(on_umock_c_error), "umock_c_init failed");
     ASSERT_ARE_EQUAL(int, 0, umocktypes_stdint_register_types(), "umocktypes_stdint_register_types failed");
 
-    REGISTER_UMOCK_ALIAS_TYPE(LPSYSTEM_INFO, void*);
+    REGISTER_UMOCK_ALIAS_TYPE(WORD, uint16_t);
+    REGISTER_UMOCK_ALIAS_TYPE(DWORD, uint32_t);
```

Comment:
I think there is a helper that registers windows types, you should use that one.